api/conf/conf.yaml

```diff
@@ -15,12 +15,21 @@
 # limitations under the License.
 #
 
+# yamllint disable rule:comments-indentation
 conf:
-  listen:                 # yamllint disable rule:comments-indentation
```

Comment:
If remove this, Will yaml lint report an error?

Reply:
@gxthrj  not remove it, just move it to other place.